clusters/nerc-ocp-infra/kustomization.yaml

```diff
@@ -11,7 +11,6 @@ resources:
   - vault
   - acm
   - xdmod
-  - loki
```

Comment:
Let's be sure to keep loki on the infra cluster for now, in addition to loki on `nerc-ocp-obs`.